### ISSUE TITLE
More than a one ldap server

### ISSUE THREAD

**Issue by topesantos** [salesagility/SuiteCRM#5107]
#### Issue
I have more than a ldap server and I don't know if all are uptime

#### Expected Behavior
Allow SuiteCRM to handle more than one ldap server

#### Possible Fix
               // this servers are in $GLOBALS['ldap_config']->settings['ldap_hostname'];
               // and can pass to an array
		$adServers=array("server1","server2","server3"); 

		$i=0;
		do {
			$ldap = ldap_connect($adServers[$i]);
			$i++;
		}while(!$ldap);

**Comment by pgorod** [comment]
I agree it's useful, LDAP shouldn't be "single point of failure" that leaves the entire company unable to log in to other systems.
